webui/react/src/utils/types.ts

```diff
@@ -80,9 +80,9 @@ export const runStateToLabel: {[key in RunState]: string} = {
   [RunState.Deleted]: 'Deleted',
   [RunState.Errored]: 'Errored',
   [RunState.Paused]: 'Paused',
-  [RunState.StoppingCanceled]: 'StoppingCanceled',
-  [RunState.StoppingCompleted]: 'StoppingCompleted',
-  [RunState.StoppingError]: 'StoppingErrored',
+  [RunState.StoppingCanceled]: 'Canceling',
+  [RunState.StoppingCompleted]: 'Completing',
+  [RunState.StoppingError]: 'Erroring',
```

Comment:
this one sounds a bit off to me :thinking: not sure what would be better though

Reply:
Yeah haha. I thought about `Failing` but felt `Erroring` is more consistent with the rest of the patterns.